config/routes.rb

```diff
@@ -71,4 +71,8 @@
       resources :ingest, only: [:create]
     end
   end
+
+  # Legacy URL support
+  # @todo what are all the URLs supported by Sufia/Scholarsphere-3?
+  get '/concern/generic_works/:id', to: 'legacy_urls#v3'
```

Comment:
@rschenk I think collections are the only other resources that have/need resolvable urls.